storey/table.py

```diff
@@ -1064,6 +1076,9 @@ def __init__(self, max_value=None, set_data=None, set_time=None):
     def aggregate(self, time, value):
         self._set_value(self.value + 1)
 
+    def aggregate_lua_script(self, vl1, vl2):
+        return f'{vl1}+{vl2}'
```

Comment:
This doesn't look like the right implementation for count. Unless I'm missing something, we should add a test that catches this.

Reply:
similar explanation to SqrValue()